src/config/clientEnv.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from 'zod'
+
+const envClientSchema = z.object({
+  API_BASE_URL: z.string().url(),
+  ASSOCIATED_DOMAIN: z.string().url(),
+  SENTRY_DSN: z.string().url(),
+  OAUTH_BASE_URL: z.string().url(),
+  IOS_GOOGLE_API_KEY: z.string(),
+  ANDROID_GOOGLE_API_KEY: z.string(),
+  OAUTH_CLIENT_ID: z.string(),
+  FB_API_KEY: z.string(),
+  FB_PROJECT_ID: z.string(),
+  FB_SENDER_ID: z.string(),
+  FB_APP_ID: z.string(),
+  FB_MEASUREMENT_ID: z.string(),
+  ENVIRONMENT: z.enum(['staging', 'production']),
+})
+
+const unParsedEnvClient = {
```

Comment:
```suggestion
const unParsedClientEnv = {
```

Sinon c'est en franglish, mais c'est pour dire qqch :D